data-structures-stacks/insert-under-top.js

```diff
@@ -0,0 +1,21 @@
+/* exported insertUnderTop */
+
+function insertUnderTop(stack, value) {
+  const placeholder = [];
+  let item;
+  for (let i = 0; i < 2; i++) {
```

Comment:
This can be done without a loop. If you have the time, try refactoring to work without a loop